custom_components/smartthinq_sensors/sensor.py

```diff
@@ -792,3 +836,28 @@ def extra_state_attributes(self):
         data.update(features)
 
         return data
+
+class LGEMicrowaveDeviceSensor(LGESensor):
+    """A sensor to monitor LGE Microwave devices"""
+
+    _wrap_device: LGEMicrowaveDevice
+
+    def __init__(
+        self,
+        api: LGEDevice,
+        description: ThinQSensorEntityDescription,
+    ):
+        """Initialize the sensor."""
+        super().__init__(api, description, LGEMicrowaveDevice(api))
+
+    @property
+    def extra_state_attributes(self):
+        """Return the optional state attributes."""
+        if not self._is_default:
+            return super().extra_state_attributes
+
+        data = {}
+        features = self._wrap_device.get_features_attributes()
+        data.update(features)
+
+        return data
```

Comment:
I think also this is useless, `LGESensor` should already implement this logic for `DEFAULT_SENSOR`